### PR TITLE
Botolytics Quickstart

### PR DESCRIPTION
**Pull request by sjyothi54**
## Description:
Raising this PR inorder to address all the comments mentioned in the PR: https://github.com/newrelic/newrelic-quickstarts/pull/1780
Ticket URL:https://issues.newrelic.com/browse/NR-150069
## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: install/botolytics/install.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+id: botolytics
+name: botolytics
+title: Bot'o'lytics
```
Comment (mdumpati) [inline install/botolytics/install.yml:3]:
```suggestion
title: Botolytics
```

File: quickstarts/botolytics/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+id:
+slug: botolytics
+title: Bot'o'lytics
```
Comment (mdumpati) [inline quickstarts/botolytics/config.yml:3]:
```suggestion
title: Botolytics
```

File: data-sources/botolytics/config.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+id: botolytics
+displayName: Bot'o'lytics
```
Comment (mdumpati) [inline data-sources/botolytics/config.yml:3]:
```suggestion
displayName: Botolytics
```

**Review by mdumpati — CHANGES_REQUESTED** [review]
@sjyothi54 Please change Bot'o'lytics to Botolytics everywhere.

File: quickstarts/botolytics/config.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  - Heiko Specht (New Relic)
+  - Linda Weimannn (New Relic)
+keywords:
+  - marketing
+  - bot
+  - traffic analysis
+  - bots
+  - fraud detection
+  - botolytics
+  - boto'o'lytics
```
Comment (mdumpati) [inline quickstarts/botolytics/config.yml:24]: No need this line

File: quickstarts/botolytics/config.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
+id:
```
Comment (mdumpati) [inline quickstarts/botolytics/config.yml:1]:
```suggestion
id:
```
Please mention id here or remove

**Review by mdumpati — CHANGES_REQUESTED** [review]
@sjyothi54 check my comments

File: quickstarts/botolytics/config.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+icon: logo.png
+level: Verified
+authors:
+  - Heiko Specht (New Relic)
+  - Linda Weimannn (New Relic)
+keywords:
+  - marketing
+  - bot
+  - traffic analysis
+
```
Comment (mdumpati) [inline quickstarts/botolytics/config.yml:21]: @sjyothi54 add apm and addData keyword here.

**Review by mdumpati — CHANGES_REQUESTED** [review]

File: install/botolytics/install.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+id: botolytics
```
Comment (brammerl) [inline install/botolytics/install.yml:1]: Please delete this file as instlalPlans are deprecated

File: quickstarts/botolytics/config.yml [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+level: Verified
+authors:
+  - Heiko Specht (New Relic)
+  - Linda Weimannn (New Relic)
+keywords:
+  - marketing
+  - bot
+  - traffic analysis
+  - apm
+  - NR1_addData
```
Comment (brammerl) [inline quickstarts/botolytics/config.yml:22]:
```suggestion
```

Comment (brammerl) [reply quickstarts/botolytics/config.yml:22]: We don't want to add quickstarts to add data unless explicitly approved by a PM and Jumpstart team.

File: quickstarts/botolytics/config.yml [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+documentation:
+  - name: Installation docs
+    url: https://docs.newrelic.com/introduction-apm
+    description: Add APM data to use this Quickstart
+installPlans:
+  - botolytics
```
Comment (brammerl) [inline quickstarts/botolytics/config.yml:29]:
```suggestion

```

File: quickstarts/botolytics/config.yml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+dashboards:
+  - botolytics
+
+dataSourceIds:
+  - botolytics
```
Comment (brammerl) [inline quickstarts/botolytics/config.yml:35]:
```suggestion
  - guided-install
```

File: data-sources/botolytics/config.yml [hunk 11]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (brammerl) [inline data-sources/botolytics/config.yml:1]: Since we're deciding to go with guided install data source we can also delete this.

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Pull request merged**